src/stores/room-list/previews/MessageEventPreview.ts

```diff
@@ -55,7 +55,11 @@ export class MessageEventPreview implements IPreview {
         }
 
         if (hasHtml) {
-            body = sanitizedHtmlNodeInnerText(body);
+            body = sanitizeHtml(body, {
+                // only allow italics, underlined, code, strikethrough
+                // and bold text in message previews
+                allowedTags: ['i', 'u', 'code', 'del', 'strong'],
```

Comment:
I am sorry I didn't realize a little detail here :man_facepalming:  We should use `<em>` instead of `<i>`. 

This is what text converted from `MD` to `HTML` looks like:

![Screenshot_20210327_130701](https://user-images.githubusercontent.com/25768714/112720216-53478c00-8efd-11eb-90c2-bafc419dbe29.png)

Reply:
Oh yeah. 🤦‍♂️
Okay, I'll make it `em`.